src/main/kotlin/com/hibob/academy/employeeFeedback/filter/AuthenticationFilter.kt

```diff
@@ -26,9 +30,12 @@ class AuthenticationFilterEmployee : ContainerRequestFilter {
         val claims = verify(cookie, requestContext)
 
         claims?.let {
-            requestContext.setProperty("companyId", it["companyId"])
-            requestContext.setProperty("employeeId", it["employeeId"])
-            requestContext.setProperty("role", it["role"])
+            val companyId = (it["company_id"] as? Number)?.toLong()
+                ?: throw WebApplicationException("Company id not found", Response.Status.UNAUTHORIZED)
```

Comment:
```suggestion
                ?: throw NotAuthorizedException("Company id not found", Response.Status.UNAUTHORIZED)
```
same for the other one too